### ISSUE TITLE
`bool_assert_comparison` improve the suggestion to include the expression 

### ISSUE THREAD

**Issue by xFrednet** [rust-lang/rust-clippy#7598]
The lint message for the  [`bool_assert_comparison`](https://rust-lang.github.io/rust-clippy/master/index.html#bool_assert_comparison) lint currently look like this:

```rs
warning: used `assert_eq!` with a literal bool
 --> src/main.rs:5:5
  |
5 |     assert_eq!(some_bool_expression, false);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: replace it with: `assert!(..)`
```

I think the suggestion can be improved to include the actual expression that should be put in the `assert!` instead of just having the placeholder `..`. 

The text for the suggestion can be retrieved with functions from [`clippy_utils::source`](https://github.com/rust-lang/rust-clippy/blob/master/clippy_utils/src/source.rs).
